internal/generate/olm-catalog/csv_go_test.go

```diff
@@ -119,13 +123,21 @@ func TestGoCSVNewWithInputsToOutput(t *testing.T) {
 
 	// Read expected CSV
 	expBundleDir := filepath.Join("expected-catalog", OLMCatalogChildDir, testProjectName, csvVersion)
-	csvExp := string(readFile(t, filepath.Join(expBundleDir, csvFileName)))
 
 	// Read generated CSV from outputDir path
 	outputBundleDir := filepath.Join(outputDir, OLMCatalogChildDir, testProjectName, csvVersion)
-	csvOutput := string(readFile(t, filepath.Join(outputBundleDir, csvFileName)))
 
-	assert.Equal(t, csvExp, csvOutput)
+	csvOutputStruct, err := getCSVFromFile(filepath.Join(outputBundleDir, csvFileName))
+	if err != nil {
+		t.Errorf("error occured %v", err)
+	}
+	projutil.RemoveSDKstampsFromCSV(csvOutputStruct)
+
+	expStruct, err := getCSVFromFile(filepath.Join(expBundleDir, csvFileName))
+	if err != nil {
+		t.Errorf("error occured %v", err)
+	}
+	assert.Equal(t, expStruct, csvOutputStruct)
```

Comment:
Manipulating string and removing contents in YAML was causing error. This verbose code could be removed once we have these tests in Ginkgo/gomega format.